src/components/activity-page/activity-page-content.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { PrimaryEmbeddable } from "./primary-embeddable";
+import { SecondaryEmbeddable } from "./secondary-embeddable";
+import { BottomButtons } from "./bottom-buttons";
+import { PageLayouts, EmbeddableSections } from "../../utilities/activity-utils";
+
+import './activity-page-content.scss';
+
+interface IProps {
+  isFirstActivityPage: boolean;
+  isLastActivityPage: boolean;
+  onPageChange: (page: number) => void;
+  page: any;
+  pageNumber: number;
+  totalPreviousQuestions: number;
+}
+
+interface IState {
+  scrollOffset: number;
+}
+
+export class ActivityPageContent extends React.PureComponent <IProps, IState> {
+  private divRef: HTMLDivElement | null;
+  public constructor(props: IProps) {
+    super(props);
+    this.state = {
+      scrollOffset: 0
+    };
+  }
+
+  render() {
+    const { isFirstActivityPage, isLastActivityPage, page, totalPreviousQuestions } = this.props;
+    const { scrollOffset } = this.state;
+    const fullWidth = page.layout === PageLayouts.Responsive;
+    const vertical = page.layout === PageLayouts.FullWidth;
```

Comment:
A priori, it seems a bit odd that `PageLayouts.FullWidth` does not determine the `fullWidth` value. Perhaps a comment to clarify the contextual meanings of the terms here?

Reply:
Agreed.  The existing layout names don't, in my opinion, describe what they do (full width layout actually has a static, constrained width and would be better named "stacked").  I renamed the local variable and added a comment to help clarify.